tools/fix-metal-includes.sh

```diff
@@ -23,11 +23,12 @@ KERNEL_LIST=" \
 arg_reduce.metal \
 conv.metal \
 gemv.metal \
+layer_norm.metal \
 random.metal \
 rms_norm.metal \
-layer_norm.metal \
```

Comment:
I just reordered these to match the CMakeLists (to make it easier to visually compare)